services/cards-consultation/src/main/java/org/opfab/cards/consultation/repositories/UserUtilitiesCommonToCardRepository.java

```diff
@@ -286,13 +284,17 @@ private List<Criteria> getFilterCriteria(CardsFilter filter) {
         filter.filters().forEach(columnFilter -> {
             if (!SPECIAL_PARAMETERS.contains(columnFilter.columnName()) && columnFilter.operation() == null) {
                 // Multiple conditions operations are not supported yet
-                criteria.add(getMatchingCriteria(columnFilter));
+                try {
+                    criteria.add(getMatchingCriteria(columnFilter));
+                } catch (NumberFormatException | NoSuchFieldException | SecurityException e) {
+                    // Ignore criteria on wrong columns
+                }
```

Comment:
I suggest handling exceptions within each case of the getMatchingCriteria switch statement and adding a meaningful `log.warn`.

 This would make it easier to identify and resolve potential problems.

Reply:
After discussion it was decided to keep it as it is, because the behaviour is similar to the current implementation: if a wrong column is specified in the condition than the condition is ignored.